presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,89 @@
+* Fix element representation in arrays returned from `ResultSet.getArray`, making it consistent with `ResultSet.getObject`.
+  Previously the elements were represented using internal client representation (e.g. `String`). ({issue}`6048`)
+* Fix `ResultSetMetaData.getColumnType` for `timestamp with time zone`. Previously the type was miscategorized as `java.sql.Types.TIMESTAMP`. ({issue}`6251`)
+* Fix `ResultSetMetaData.getColumnType` for `time with time zone`. Previously the type was miscategorized as `java.sql.Types.TIME`. ({issue}`6251`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned in the `ResultSet`. ({issue}`6240`)
+  
+## CLI Changes
+
+* Fix rendering of `row` values with unnamed fields. Previously they were printed using fake field names like `field0`, `field1`, etc. ({issue}`4587`)
+* Fix query progress reporting. ({issue}`6119`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned to the client. ({issue}`6238`)
+
+## Hive Connector Changes
+
+* Allow configuring S3 endpoint in security mapping. ({issue}`3869`)
+* Add support for S3 streaming uploads. Data is uploaded to S3 as it is written, rather
+  than staged to a local temporary file. This feature is disabled by default, and can be enabled
+  using the `hive.s3.streaming.enabled` configuration property. ({issue}`3712`, {issue}`6201`)
+* Decrease default refresh thread count for metastore cache. ({issue}`6156`)
+* Reduce load on metastore when background cache refresh is enabled. Partition metadata and statistics are 
+  no longer refreshed in the background. ({issue}`6101`)
+* Verify that data is in the correct bucket file when reading bucketed tables.
+  This is enabled by default, as incorrect bucketing can cause incorrect query results,
+  but can be disabled using the `hive.validate-bucketing` configuration property
+  or the `validate_bucketing` session property. ({issue}`6012`)
+* Allow fallback to legacy Hive view translation logic via `hive.legacy-hive-view-translation` config property or 
+  `legacy_hive_view_translation` session property. ({issue}`6195 `)
+* Add deserializer class name to split information exposed to the event listener. ({issue}`6006`)
+* Improve performance when querying tables that contain symlinks. ({issue}`6158`, {issue}`6213`)
+    
+## Iceberg Connector Changes
+
+* Improve performance of queries containing filters on non-partition columns. Such filters are now used 
+  for optimizing split generation and table scan.  ({issue}`4932`)
+* Add support for Google Cloud Storage and Azure Storage. ({issue}`6186`)
+
+## Kafka Connector Changes
+
+* Allow writing `timestamp with time zone` values into columns using `milliseconds-since-epoch` or 
+  `seconds-since-epoch` JSON encoders. ({issue}`6074`)
+
+## Other Connector Changes
+
+* Fix table metadata caching for PostgreSQL, MySQL, SQL Server, Redshift, MemSQL and Phoenix connectors. ({issue}`6081`, {issue}`6167`)
```

Comment:
It would be nice to capture shortly what is the nature of fix? Was cache not working? Or misbehave? Can it be done using single sentence?

Reply:
I changed it to "Fix ineffective table metadata caching..."